projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-cart.ts

```diff
@@ -23,6 +23,23 @@ export function clickOnEditConfigurationLink(cartItemIndex: number): void {
     });
 }
 
+/**
+ * Clicks on the 'Edit Configuration' link in add-to-cart dialog.
+ *
+ */
+export function clickOnEditConfigurationLinkInAddToCartDialog(): void {
```

Comment:
do you think it would make sense to move this into the common helper, where also some other functions of the add-to-cart dialog are placed?

Reply:
Agree! Done!